pybv/io.py

```diff
@@ -494,8 +494,8 @@ def _write_bveeg_file(eeg_fname, data, orientation, format, resolution, units):
 
     # Swap bytes if system architecure is big-endian and dtype is "native" or
     # "big-endian"
-    byteorder = data.dtype.byteorder
-    if (sys.byteorder == "big" and byteorder == "=") or byteorder == ">":
+    byteorder = data.dtype.byteorder  # pragma: no cover
+    if (sys.byteorder == "big" and byteorder == "=") or byteorder == ">":  # pragma: no cover  # noqa: E501
```

Comment:
The first pragma is not needed (that line is covered), and I removed one `#` which I think is also not needed

```suggestion
    byteorder = data.dtype.byteorder
    if (sys.byteorder == "big" and byteorder == "=") or byteorder == ">":  # pragma: no cover noqa: E501
```

Reply:
Why don't we cover this with a test? We just need to create a `data` array with `dtype=">f8"` or something.

Reply:
ohhh, sure. I'll write a small test for it as well

Reply:
Also, let me know what the merging strategy is. I could squash and tidy up the commits if you want.

Reply:
No worries, we'll squash before merging.

Reply:
Added in a small test. Also rebased the main branch of my fork to see if the travis tests work or not. They hopefully should work.